filters/builtin/comment.go

```diff
@@ -0,0 +1,23 @@
+package builtin
+
+import "github.com/zalando/skipper/filters"
+
+type comment struct{}
+
+// NewComment is a filter to comment a filter chain. It does nothing
+func NewComment() filters.Spec {
+	return &comment{}
+}
+
+func (*comment) Name() string {
+	return filters.CommentName
+}
+
+func (c *comment) CreateFilter(args []interface{}) (filters.Filter, error) {
+	return c, nil
```

Comment:
Makes sense to restrict args to single string value

Reply:
why limit?

Reply:
because filter has a purpose, I dont think its a good idea to allow `comment("foo", "bar", "baz")` or `comment(1, 2, 3)`

Reply:
Maybe someone wants to write:
```
comment(
1, "foo bar",
2, "baz",
)
```
Why should we care about the input?